frontend/src/components/molecules/messages/Message.tsx

```diff
@@ -186,9 +186,12 @@ const Message = memo(
                       {actions?.length ? (
                         <MessageActions message={message} actions={actions} />
                       ) : null}
-                      {scorableRun && isScorable ? (
-                        <MessageButtons message={message} run={scorableRun} />
-                      ) : null}
+                      <MessageButtons
```

Comment:
Previously, feedback buttons were displayed if the run ended with an assistant message. Feedback buttons should be displayed at the end of the run regardless of the type of the last step